include/picongpu/fields/incidentField/Solver.kernel

```diff
@@ -158,16 +155,13 @@ namespace picongpu
                         auto const incidentFieldShift1 = incidentFieldShiftBase + inCellShift1;
                         auto const incidentFieldShift2 = incidentFieldShiftBase + inCellShift2;
                         auto const updatedFieldShift = (updatedGridIdx - huygensSurfaceIdx)[T_axis];
-                        updatedField(updatedGridIdx) += getUpdatedFieldCorrection(
+                        destField(updatedGridIdx) += getUpdatedFieldCorrection(

```

Comment:
The refactoring that the field is passed into the function instead of storing as a member is required because for unknown reasons the field box was calling the const version of the memory access even if the object was not defined const.

Reply:
My fault, the problem was that the data box was stored as a member of the functor. I called the const version of the functor and therefore all members are seen as const too.